stats/src/test/java/io/airlift/stats/cardinality/BenchmarkDenseHll.java

```diff
@@ -63,6 +72,27 @@ public void initialize()
         }
     }
 
+    @State(Scope.Thread)
+    public static class MergeData
+    {
+        public DenseHll base;
+        public DenseHll toMerge;
+
+        @Setup(Level.Iteration)
+        public void initialize()
+        {
+            base = new DenseHll(12);
+            for (int i = 0; i < 1_000_000; i++) {
```

Comment:
Is it important that here are `1_000_000` and below `100` only? Should that be parametrized?

Reply:
No special meaning other than having a "large" and a "small" set for the comparison with sparse hll in the next commit to be meaningful (sparse can store only a small set).